### ISSUE TITLE
Hello , can you tell me how to use with remote sensing 

### ISSUE THREAD

**Issue by ENjoy924** [priyankshah7/scikit-hyper#38]
hello , I`m very interesting in your tool and this can solve my issue in hyperspectral image classification.The dataset is IndianPines and Salinas,but I don`t know how to use in this dataset because your example is random data.Can you give me a example in IndianPines dataset,thank you very much.

**Comment by priyankshah7** [comment]
I'll update the README and docs to include an example using one of those datasets in the next release. For now, here's an example using the IndianPines dataset downloaded from [here](https://purr.purdue.edu/publications/1947/1). 

```python
import numpy as np
from skimage import io
import matplotlib.pyplot as plt

from skhyper.process import Process
from skhyper.cluster import KMeans
from skhyper.svm import SVC

# Reading in the data from Site3 in the IndianPines dataset
im = io.imread('19920612_AVIRIS_IndianPine_Site3.tif')
```

The data for `scikit-hyper` needs to be in the form `[x, y, (z), spectra]` so:

```python
im_transpose = np.transpose(im)
X = Process(im_transpose, scale=True)

# Clustering using k-means
mdl_kmeans = KMeans(n_clusters=5)
mdl_kmeans.fit(X)
```

For classification, you'll need the ground truth labels. For simplicity, I'll just use one of the tif files in the dataset from the link above:

```python
im_label = io.imread('19920612_AVIRIS_IndianPine_Site3_gr.tif')

# Using support vector machines for classification
mdl_svc = SVC()
mdl_svc.fit(X, im_label)
```

To classify new data from the trained model above, you use the normal methods from scikit-learn, i.e. `predict`, `predict_proba` or `predict_log_proba`. I'll just use a random dataset here for the new dataset for now (and amend for a proper example later):

```python
# Random dataset with the same dimensions as the one used to train the model
x_random = np.random.rand(145, 145, 220)

x_classified = mdl_svc.predict(x_random)
```

Hopefully that helps. As mentioned, I'll put a thorough example in the next update.

**Comment by priyankshah7** [comment]
If you do have any suggestions or need help let me know, but please note that some features are subject to changes which won't be backward compatabile (at least until version 0.1 is released).

**Comment by priyankshah7** [comment]
I'm closing this now, if you have any further issues please feel free to open it again.
